controllers/spacerequest/spacerequest_controller_test.go

```diff
@@ -310,6 +310,61 @@ func TestCreateSpaceRequest(t *testing.T) {
 				HasTier(sr.Spec.TierName).
 				HasParentSpace("jane") // the parent space is set as expected
 		})
+
+		t.Run("subSpace target cluster is different from spacerequest cluster", func(t *testing.T) {
+			// given
+			parentSpaceWithTarget := spacetest.NewSpace(test.HostOperatorNs, "jane", spacetest.WithSpecTargetCluster("member-2"))
```

Comment:
I did bad copy-paste :man_facepalming: 
it should be member-1 when subspace goes to member-2
```suggestion
			parentSpaceWithTarget := spacetest.NewSpace(test.HostOperatorNs, "jane", spacetest.WithSpecTargetCluster("member-1"))
```

Reply:
And what's even worst, that I've blindly copied your code 🤣 🤦‍♂️ 

Hopefully is friday :)